website/cue/reference/components/transforms/base/route.cue

```diff
@@ -14,14 +14,24 @@ base: components: transforms: route: configuration: route: {
 	required: false
 	type: object: options: "*": {
 		description: """
-			A table of route identifiers to logical conditions representing the filter of the route.
+			An event matching condition.
```

Comment:
It looks like this description change is losing some important details. Notably, what are the keys in this context? What happens about unmatched events, or the `_default` route?

Reply:
Yep, agreed. I'll figure this one out. Good catch.

Reply:
Ah, sorry, right: the description being removed is already specified at the top level.

So if you look at https://vector.dev/docs/reference/configuration/transforms/route/#route, the same description is specified for `route` and `route.*`. Realistically, it only makes sense on one or the other... which is why we're not doing that here.

However, the way it's pulling in the docs for `AnyCondition` and using them for `route.*` also isn't what I want, so I'll fix that up.

Reply:
Addressed this in 991dfb25.